src/main/java/com/minecolonies/coremod/colony/requestsystem/requests/StandardRequests.java

```diff
@@ -343,11 +343,9 @@ protected AbstractCraftingRequest(
         public final ITextComponent getShortDisplayString()
         {
             final ITextComponent result = new NonSiblingFormattingTextComponent();
-            final ITextComponent preType = new TranslationTextComponent(getTranslationKey());
 
-            result.appendSibling(preType);
-
-            preType.appendSibling(getRequest().getStack().getTextComponent());
+            result.appendSibling(new StringTextComponent(getRequest().getMinCount() + " * Recipe:"));
+            result.appendSibling(getRequest().getStack().getTextComponent());
```

Comment:
This makes no sense.
What are you trying to achieve here?

Reply:
Also not localizable!

Reply:
Updated to be localizable, this now produces a string like "8 * Recipe: [Plain Oak Timber Frame]" for the crafting job entry in the request tree.